mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.kt

```diff
@@ -467,14 +467,8 @@ abstract class ContentController protected constructor(private val activity: Mai
         return false
     }
 
-    override fun closeFragment() {
-        if (temporaryPage != null) {
-            temporaryPage = null
-            activity.updateTitle()
-            updateActionBarState()
-            updateFragmentState(FragmentUpdateReason.PAGE_UPDATE)
-            updateConnectionState()
-        }
```

Comment:
This code needs to be restored otherwise the action bar doesn't show up when switching from MainUI to Sitemaps.